discovery-infra/tests/test_cancel_and_reset.py

```diff
@@ -0,0 +1,113 @@
+import os
+import pytest
+
+import assisted_service_api
+import utils
+import consts
+
+if os.environ.get('NODE_ENV') == 'QE_VM':
+    import node_controlers.qe_vm_controler
+
+ssh_public_key = os.environ.get('SSH_PUB_KEY')
+remote_service_url = os.environ.get('REMOTE_SERVICE_URL')
+pull_secret = os.environ.get('PULL_SECRET')
+cluster_name = os.environ.get('CLUSTER_NAME')
+offline_token = os.environ.get('OFFLINE_TOKEN')
+openshift_version = os.environ.get('OPENSHIFT_VERSION')
+base_domain = os.environ.get('BASE_DOMAIN')
+iso_download_path = os.environ.get('ISO')
+num_masters = int(os.environ.get('NUM_MASTERS'))
+num_workers = int(os.environ.get('NUM_WORKERS'))
+
+nodes_count = num_masters + num_workers
+
+api_client = assisted_service_api.create_client(remote_service_url)
+controler = node_controlers.qe_vm_controler.create_controler()
+
+@pytest.mark.regression
+def test_cancel_reset_after_node_boot():
+
+# Define new cluster
```

Comment:
Shouldn't all tests have setup(), and teardown ()functions?
I also assume that at least for teardown() we should have some common, default, cleanup functionality

Reply:
Adding, Moved the PR to draft. Will reopen when setup() and teardown() is in place

Reply:
added, setup and tear down in the fixtures